Kickstarter-iOS/Views/Controllers/EditorialProjectsViewController.swift

```diff
@@ -217,6 +239,18 @@ public final class EditorialProjectsViewController: UIViewController {
   @objc private func closeButtonTapped() {
     self.viewModel.inputs.closeButtonTapped()
   }
+
+  private func applyViewTransforms(withY y: CGFloat) {
```

Comment:
These values are somewhat arbitrary which is why I just did the one output from the VM and I'm not really testing them... Maybe I'm just lazy, I'm just not sure what we'd gain from testing them necessarily. But if there's a desire to do so I can add tests.